cart/src/test/java/com/yas/cart/service/CartServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package com.yas.cart.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.anyList;
+import static org.mockito.Mockito.when;
+
+import com.yas.cart.model.Cart;
+import com.yas.cart.repository.CartItemRepository;
+import com.yas.cart.repository.CartRepository;
+import com.yas.cart.viewmodel.CartItemVm;
+import com.yas.cart.viewmodel.CartListVm;
+import com.yas.commonlibrary.exception.BadRequestException;
+import com.yas.commonlibrary.exception.NotFoundException;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+class CartServiceTest {
+
+    @Mock
+    private CartRepository cartRepository;
+    @Mock
+    private CartItemRepository cartItemRepository;
+    @Mock
+    private ProductService productService;
+    @InjectMocks
+    private CartService cartService;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
```

Comment:
`@RunWith(MockitoJUnitRunner.class)` already handle the mock initialization. You don't need to call `openMocks` manually.